.github/workflows/rust.yml

```diff
@@ -127,18 +130,29 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        rv: [1.57.0, stable]
+        #rv: [1.57.0, stable]
```

Comment:
remove?

Reply:
Remove.

Reply:
On a related note, builds on 1.57 are currently required which prevents this PR from being merged.